src/main/org/nlogo/app/ModelsLibraryDialog.java

```diff
@@ -32,11 +34,6 @@ public static String open(java.awt.Frame parent)
       me = new ModelsLibraryDialog(parent);
       me.tree.setSelectionRow(0);
     }
-    // not sure I really understand why this should be necessary
-    // (it isn't on Windows), but at least on Macs, this ensures
-    // that the tree gets the keyboard focus when the dialog opens
-    // - ST 12/17/04
-    me.setFocusable(false);
```

Comment:
I'm afraid I don't remember anything about this

I do like using the arrow keys to navigate the tree, but I'm fine w/ the change.